eth/stubclient.go

```diff
@@ -213,28 +212,35 @@ type StubClient struct {
 	TranscoderPoolError          error
 	RoundLocked                  bool
 	RoundLockedErr               error
+	Errors                       map[string]error
 }
 
 type stubTranscoder struct {
 	ServiceURI string
 }
 
 func (e *StubClient) Setup(password string, gasLimit uint64, gasPrice *big.Int) error { return nil }
-func (e *StubClient) Account() accounts.Account                                       { return accounts.Account{Address: e.TranscoderAddress} }
-func (e *StubClient) Backend() (Backend, error)                                       { return nil, nil }
+func (e *StubClient) Account() accounts.Account {
+	return accounts.Account{Address: e.TranscoderAddress}
+}
+func (e *StubClient) Backend() (Backend, error) { return nil, nil }
 
 // Rounds
 
 func (e *StubClient) InitializeRound() (*types.Transaction, error) { return nil, nil }
-func (e *StubClient) CurrentRound() (*big.Int, error)              { return big.NewInt(0), e.RoundsErr }
-func (e *StubClient) LastInitializedRound() (*big.Int, error)      { return e.Round, e.RoundsErr }
+func (e *StubClient) CurrentRound() (*big.Int, error)              { return big.NewInt(0), e.Errors["CurrentRound"] }
+func (e *StubClient) LastInitializedRound() (*big.Int, error) {
+	return e.Round, e.Errors["LastInitializedRound"]
+}
 func (e *StubClient) BlockHashForRound(round *big.Int) ([32]byte, error) {
-	return e.BlockHashToReturn, nil
+	return e.BlockHashToReturn, e.Errors["BlockHashForRound"]
 }
-func (e *StubClient) CurrentRoundInitialized() (bool, error)    { return false, nil }
-func (e *StubClient) CurrentRoundLocked() (bool, error)         { return e.RoundLocked, e.RoundLockedErr }
-func (e *StubClient) CurrentRoundStartBlock() (*big.Int, error) { return nil, nil }
-func (e *StubClient) Paused() (bool, error)                     { return false, nil }
+func (e *StubClient) CurrentRoundInitialized() (bool, error) { return false, nil }
+func (e *StubClient) CurrentRoundLocked() (bool, error)      { return e.RoundLocked, e.RoundLockedErr }
```

Comment:
Since we don't call this method in the `TimeWatcher` I decided to keep the diff small for now, but can definitely submit a follow up PR to clean up the rest of the errors in the stub Eth client to follow the same principle.